src/schema/objects/columns.yaml

```diff
@@ -302,13 +302,11 @@ notch:
   display_name: Notch frequencies
   description: |
     Frequencies used for the notch filter applied to the channel, in Hz.
+    If the low and high frequencies are supplied in addition to the center
+    frequency, it is RECOMMENDED that they take the form `[low, center, high]`,
+    for example `[60, 120, 180]`.
     If no notch filter applied, use `n/a`.
-  anyOf:
-    - type: number
-      unit: Hz
-    - type: string
-      enum:
-        - n/a
+  type: string
```

Comment:
Is there anyway to include a regex pattern like it does below? I feel like that could be useful for the validator too

Reply:
Yes, but it would be ugly. You need to match `SQUARE_LEFT SPACE NUMBER COMMA SPACE NUMBER COMMA SPACE NUMBER SQUARE_RIGHT`

And `NUMBER` is 

https://github.com/bids-standard/bids-specification/blob/eff64847adf07638ab4eb39cacb31e193c13bc43/src/schema/objects/formats.yaml#L26-L30

And you probably want to be forgiving and allow extra spaces around the numbers, so you really want to replace `SPACE NUMBER` with `SPACE* NUMBER SPACE*`.